### PR TITLE
Update intro-to-cryptography.md

### PR DESCRIPTION
**Pull request by bunnyBites**
Fixed typos and improved sentence delivery @jamesrp13

### CODE REVIEW

**Comment by jamesrp13** [comment]
Thanks @bunnyBites!

@mikemaccana I'll defer to you on this one since you write this but the changes seem good to me.

**Comment by bunnyBites** [comment]
> Thanks @bunnyBites!
> 
> @mikemaccana I'll defer to you on this one since you write this but the changes seem good to me.

Thanks 🤓. The documentation is great, will go through the course and see to make it even better🫡.

**Review by mikemaccana — APPROVED** [review]
Thankyou!

File: content/intro-to-cryptography.md [hunk 1]
```diff
@@ -29,20 +29,21 @@ There's a variety of symmetric cryptography algorithms, but the most common you'
...
+ - Your bank card has a secret key inside it that's used to sign transactions.
 
    Your bank can confirm you made the transaction by checking them with the matching public key.
- - Websites include a public key in their certificate, your browser will use this public key to encrypt the data (like personal information, login details, and credit card numbers) it sends to the web page. 
+ - Websites include a public key in their certificate. Your browser will use this public key to encrypt the data (like personal information, login details, and credit card numbers) it sends to the web page. 
 
-   The website has the matching private key, so the website can read the data. 
- - Your electronic passport was signed by the country that issued it, to ensure the passport isn't forged. 
+   The website has the matching private key so that the website can read the data. 
+ - Your electronic passport was signed by the country that issued it to ensure it isn't forged.
```
[the remark this answers is not in the document]
Comment (mikemaccana) [reply content/intro-to-cryptography.md]: Changed it back and merged.

[the remark this answers is not in the document]
Comment (bunnyBites) [reply content/intro-to-cryptography.md]: Yeah, that is something that can happen. Thank you for taking care of it. 😊

**Pull request merged**